service/storage_proxy.cc

```diff
@@ -2785,6 +2788,14 @@ void storage_proxy_stats::stats::register_stats() {
         sm::make_total_operations("cas_dropped_prune", cas_replica_dropped_prune,
                        sm::description("how many times a coordinator did not perform prune after cas"),
                        {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(),
+
+        sm::make_total_operations("received_hint_count", received_hint_count,
+                        sm::description("number of hints received by this node"),
+                       {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(false),
+
+        sm::make_total_bytes("received_hint_total_size", received_hint_total_size,
+                        sm::description("total size of hints received by this node"),
+                       {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(false),
```

Comment:
Do we need `.set_skip_when_empty(false)` here? I suppose that the metric will not be larger than 0 on other scheduling groups than `streaming`, so this is a worthwhile optimization. On the other hand, if there are some hints received in other scheduling groups then it will be a clear indication of a bug.